src/mrack/providers/provider.py

```diff
@@ -28,6 +28,7 @@
 STRATEGY_RETRY = "retry"
 RET_CODE = 0
 HOST_OBJ = 1
+SPECS = 1
```

Comment:
Could you please leave a comment describing this as the default index used to fetch data from the response or something like it?

Reply:
Can do